### DISCUSSION TITLE
How to set MT_SERVICES

### DISCUSSION THREAD

**Discussion by urgn** [WeblateOrg/weblate#6399] [Q&A]
Hello! I want to use weblate for machine translations. I've read the documentation https://docs.weblate.org/en/latest/admin/machine.html, and It says that I should turn on the integrations using MT_SERVICES.
But I can't find this MT_SERVICES option in admin settings.
How should I set it up?
I'm running the weblate using docker-compose file.

**Comment by nijel** [comment]
When using Doker, only providing an API key or URL is needed, see https://docs.weblate.org/en/latest/admin/install/docker.html#machine-translation-settings

**Reply by urgn** [reply]
Thanks a lot! Already find it in this file https://github.com/WeblateOrg/weblate/blob/main/weblate/settings_docker.py#L808
